### ISSUE TITLE
Failing to resolve a property reference when used via local in-place reference

### ISSUE THREAD

**Issue by outbounder** [camplight/organic-dna-resolve#5]
Having the following use case

## `dna/_staigng/cells.json`

```
{
  "cwd": "something",
  "start": [
    "@_staging.http-cell.start
  ]
}
```  

## `dna/_staging/http-cell.json`

```
{
  "cwd": "@_staging.cells.cwd",
  "start": "&{cwd} && ..."
}
```

---

When used the above dna after resolving is having `&{cwd}` not resolved and has a value of `@_staging.cells.cwd`

This may not implemented due performance concerns, however if it is possible it would be nice to have especially when the performance impact is low (ns) to medium (ms) in scale :)

**Comment by multi** [comment]
Hey @outbounder 👋 
just made it [here](https://github.com/camplight/organic-dna-resolve/pull/6) :) Any comments are welcome as always :bowtie:

**Comment by outbounder** [comment]
@multi :+1: no comments :) just release it and close this issue so that I can update also https://github.com/outbounder/organic-dna-loader ;)

**Comment by outbounder** [comment]
closing, released as organic-dna-resolve@1.0.0 ;) :+1: :flight_departure: :rocket:
